newrelic/console.py

```diff
@@ -446,15 +447,15 @@ def __thread_run(self):
             listener.bind(self.__listener_socket)
         else:
             try:
-                os.unlink(self.__listener_socket)
+                Path(self.__listener_socket).unlink()
             except Exception:
                 pass
 
             listener = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
             listener.bind(self.__listener_socket)
 
             atexit.register(self.__socket_cleanup, self.__listener_socket)
-            os.chmod(self.__listener_socket, 0o600)
+            Path(self.__listener_socket).chmod(0o600)
```

Comment:
Does this work on windows?

Reply:
Sounds like no, but it'll ignore it. Can try and fix that when we get to trying to run this on Windows if it's necessary to fix.

https://docs.python.org/3.13/library/os.html#os.chmod